classes/02class/exercises/c02-network03/ramiray/README.md

```diff
@@ -0,0 +1,71 @@
+# c02-network03
+
+## Commands Execution Output
+
+- Commands for creating a new VPC:
+```
+1.Creating VPC with  deafult tenancy, cidr block and no-ipv6: 
+
+aws ec2 create-vpc --cidr-block 172.16.0.0/16 --no-amazon-provided-ipv6-cidr-block --instance-tenancy default
```

Comment:
no need to specify the last two parameters as they are default, but no harm in leaving them